src/AzureRepositories/Repositories/GasPriceRepository.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System.Numerics;
+using System.Threading.Tasks;
+using AzureStorage;
+using Core.Repositories;
+using Microsoft.WindowsAzure.Storage.Table;
+
+namespace AzureRepositories.Repositories
+{
+    public class GasPriceRepository : IGasPriceRepository
+    {
+        private const string PartitionKey = "GasPrice";
+        private const string RowKey = "GasPrice";
+
+        private readonly INoSQLTableStorage<GasPriceEntity> _table;
+
+        public GasPriceRepository(INoSQLTableStorage<GasPriceEntity> table)
+        {
+            _table = table;
+        }
+
+
+        public async Task<IGasPrice> GetAsync()
+        {
+            var entity = await _table.GetDataAsync(PartitionKey, RowKey);
+
+            if (entity != null)
+            {
+                return new GasPrice
+                {
+                    Max = BigInteger.Parse(entity.Max),
+                    Min = BigInteger.Parse(entity.Min)
+                };
+            }
+            else
+            {
+                return new GasPrice
+                {
+                    Max = 50000000000,
```

Comment:
It would break production for now.
It should be
Max 120000000000
Min 100000000000